lib/plugins/mongoose-encryption.js

```diff
@@ -321,12 +341,22 @@ var mongooseEncryption = function(schema, options) {
 
         // generate random iv
         crypto.randomBytes(IV_LENGTH, function(err, iv) {
-            var cipher, jsonToEncrypt, objectToEncrypt;
+            var cf, cipher, jsonToEncrypt, objectToEncrypt;
             if (err) {
                 return cb(err);
             }
+
+            if (cfMode === 'disable') {
+                cf = encryptedFields;
+            } else {
+                cf = that._cf || encryptedFields;
```

Comment:
When Document instance have helper _cf propety, it defines the list of encrypted fields.